### ISSUE TITLE
Deal with multiple verses.

### ISSUE THREAD

**Issue by vvasuki** [shreevatsa/sanskrit#102]
I have an issue to report about the results for the following input:
```
सुप्रभातप्रयोगेण रात्रौ लब्धे भवत्कृते।
पद्ये शङ्के स्वदेशं शं प्राप्तवन्तं‌ द्विजोत्तमम्॥

सपर्या भूसुराणां यत् कृतं दिष्ट्या ततः फलम्।
मन्येऽहं गृहिणीगर्भे वर्धमानं सविस्मयम्॥

वर्धन्तां ब्राह्मणा लोके सर्वारिष्टविनाशकाः।
पुष्यतां धार्मिको लोकः शान्तये तेजसे दिवे॥
```

yields "accidental  anuShTubh match" messages.

I'd like to check multiple verses in the toolbox at one go - currently this is tough.

**Comment by shreevatsa** [comment]
Good point; Marcis (@gasyoun) also asked for this a few times and in fact was using an old version that happened to support it better. I actually started writing some code to run on an entire long text (like from GRETIL) after some heuristics to detect breaking up verses.

I think what would be nice here is if the tool can either auto-detect that there are multiple verses, or give the user an explicit option to say "I have multiple verses here" (I'm thinking of an option like "treat input as [ ] quarter verse [ ] half verse [ ] single verse [ ] multiple verses [ ] try everything (default)"), that will help.

Will see if I can dig up this code
